src/headers/SiteHeader.scss

```diff
@@ -393,3 +393,16 @@
     width: 350px;
   }
 }
+
+.skip-link {
+  @apply sr-only;
+  &:focus {
```

Comment:
Since this is just for screen reader, I don't know how visually accessible this needs to be. But when it is in focus the site logo covers up the "Skip to main content" text 
![image](https://user-images.githubusercontent.com/42942267/215518251-5874d971-f9c4-410a-b415-a38477f0e2bb.png)

Reply:
Good catch! Updated the CSS to include a Z-index value